armi/utils/plotting.py

```diff
@@ -766,7 +766,7 @@ def plotAssemblyTypes(
         maxAssems = numAssems
 
     if yAxisLabel is None:
-        yAxisLabel = "THERMALLY EXPANDED AXIAL HEIGHTS (CM)"
+        yAxisLabel = "Thermally Expanded Axial Heights (cm)"
```

Comment:
FYI `Block.getInputHeight` may or may not be the thermally expanded height. It depends on the `inputHeightsConsideredHot` setting

Reply:
Ah, that's a fair point.  Let me take a second pass at that label.